queries/haskell/injections.scm

```diff
@@ -9,14 +9,7 @@
 
 ;; -----------------------------------------------------------------------------
 ;; shakespeare library
-;; NOTE: doesn't support templating
-
-; CoffeeScript: Text.Coffee
-(quasiquote
- (quoter) @_name
- (#eq? @_name "coffee")
- ((quasiquote_body) @coffeescript)
```

Comment:
You can also leave this commented with a todo so it gets added when a coffescript parser is added.

Reply:
Ok, I'll add it back in but comment it out